scripts/nbconverted/1_detect_nonlinear_signal.py

```diff
@@ -0,0 +1,219 @@
+
+# coding: utf-8
+
+# # Simulation experiment: nonlinear signal detection
+# 
+# **Question**: Can VAE detect nonlinear signal in simulated data?
+# 
+# **Hypothesis**: If we generate a gene expression dataset with a strong signal, step function, a nonlinear compression method (VAE) will be able to learn this signal and predict new gene expression patterns better compared to using a linear comparession method (PCA) and using no compression method (all the genes)
+#  
+# **Study design**:
+# 1. Input: Add step function to to Pa gene expression dataset:
+# 2. Approach: Train nonlinear (VAE) and linear (PCA) compression algorithms using this simulated data
+# 3. Evaluation: For each sample in the Pa dataset compare spearman corr(predicted expression, actual expression)
+# 4. Output: Boxplot of the correlation scores per sample 
```

Comment:
Is there a box plot for these multiple seed runs?

Reply:
There is eventually supposed to be boxplots created, but I wasn't able to get to it and didn't want the PR to get too big.

One boxplot will correspond to one random seed run, that correct.  I have updated my comment so its more clear.